frontend/src/sass/pages/main-landing.scss

```diff
@@ -183,7 +183,7 @@
     }  
   }
 
-  #help-pick-background{
```

Comment:
small issue but if you want to address it for code cleanliness / consistency there should be a space before the `{`